docs/actionable-messages/message-card-reference.md

```diff
@@ -11,7 +11,7 @@ localization_priority: Priority
 # Legacy actionable message card reference
 
 > [!NOTE]
-> This document describes the original JSON format for the actionable message card format. For actionable messages sent via email, this has been replaced with the [Adaptive Card format](adaptive-card.md). Microsoft recommends that new actionable message integrations use the Adaptive Card format, and existing integrations consider updating to Adaptive Card format. The Adaptive Card format is **required** to support Outlook for iOS and Outlook for Android. However, if you are sending actionable messages via an Office 365 connector, or to a Microsoft Teams connector, you must continue to use the message card format.
+> This document describes the original JSON format for the actionable message card format. For actionable messages sent via email, this has been replaced with the [Adaptive Card format](adaptive-card.md). Microsoft recommends that new actionable message integrations use the Adaptive Card format, and existing integrations consider updating to Adaptive Card format. The Adaptive Card format is **required** to support Outlook on iOS and Android. However, if you are sending actionable messages via an Office 365 connector, or to a Microsoft Teams connector, you must continue to use the message card format.
```

Comment:
Should ms.date be updated in this article?

Reply:
@kbrandl There is no ms.date on this article. Should I add one?

Reply:
@ElizabethSamuel-MSFT yes, please add it. Looks like Jason's recently started removing ms.date from articles that he edits in this repo. This doesn't negatively impact end-user experience, but CATS will report an error if ms.date is missing, so we've been including ms.date in all articles (to align with best practices as defined by CATS). @jasonjoh FYI.